src/releasers/go-yoshi.ts

```diff
@@ -95,9 +92,10 @@ export class GoYoshi extends ReleasePR {
     });
     const cc = new ConventionalCommits({
       commits: commits,
-      githubRepoUrl: this.repoUrl,
+      owner: this.gh.owner,
```

Comment:
Just standardizing on splitting own `owner` and `repo` in more places?

Reply:
the idea is to consider `repoUrl` as denormalized shorthand user input for `factory.ts` and that it should be normalized only once into `{owner, repo}` and everything else inside the release-please codebase should rely on those directly rather than either carry around this copy of the short hand that everyone then has to convert. It would be a different story if different actors converted it differently but they all just use `parse-github-repo-url` to get the owner and repo.